agent/pkg/dagent/utils/config.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"github.com/gin-gonic/gin"
+
+	"github.com/dyrector-io/dyrectorio/agent/internal/util"
+	"github.com/dyrector-io/dyrectorio/agent/pkg/dagent/config"
+)
+
+func GetConfigFromGin(c *gin.Context) *config.Configuration {
```

Comment:
GetConfigFromGinContext would be more accurate.
This should be in somewhere in the internal/util